runtime/parachains/src/scheduler/tests.rs

```diff
@@ -1033,55 +1026,42 @@ fn availability_predicate_works() {
 			_ => None,
 		});
 
-		// assign some availability cores.
+		// assign some availability core.
 		{
 			let entry_ttl = 10_000;
 			AvailabilityCores::<Test>::mutate(|cores| {
-				cores[0] =
-					CoreOccupied::Paras(ParasEntry::new(Assignment::new(chain_a), entry_ttl));
-				cores[1] =
-					CoreOccupied::Paras(ParasEntry::new(Assignment::new(thread_a), entry_ttl));
+				cores[0] = Some(CoreOccupied::Parachain);
 			});
 		}
 
-		run_to_block(1 + thread_availability_period, |_| None);
-		//assert!(Scheduler::availability_timeout_predicate().is_none());
+		run_to_block(1 + paras_availability_period, |_| None);
+		assert!(Scheduler::availability_timeout_predicate().is_none());
```

Comment:
The `availability_timeout_predicate` function no longer returns an Option.

Reply:
I noticed this, but what should I check for here since `availability_timeout_predicate` returns a closure?

Reply:
Discussed with @eskimor earlier, we should just remove the assertions in the test.